src/utils/viewconf.js

```diff
@@ -56,6 +59,35 @@ export function traverseViewConfig(viewConf, callback) {
     }
 }
 
+/**
+ * Get a HiGlass viewConfig object for a specific track.
+ * @param {object} viewConf A valid HiGlass viewConfig object.
+ * @param {string} viewId The uid of view containing the `horizontal-multivec` track that was the target of the action.
+ * @param {string} trackId The uid of the `horizontal-multivec` track that was the target of the action. 
+ * @return {object} A part of HiGlass viewConfig object for a specific track.
+ */
+export function getViewConfigOfSpecificTrack(viewConfig, viewId, trackId) {
```

Comment:
To reduce confusion, can we refer to the viewConfig as the whole big object with `{ views: [ { uid, tracks: { center, top, ... }, ... }, ... ], ... }`, and then for specific tracks or views, the "view definition" and "track definition"?

Reply:
Sounds like a great idea, and it looks better than just using `viewConfig`.

But would it be confusing with wrapper options when we use `viewConfig`, `definitions`, and `options` all together?

What do you think of using `higlassConfig` for the whole big object instead (which is certainly not a consistent name compared to how we call in the HiGlass core), and `viewConfig` and `trackConfig` for the config of a specific view and track? (Similarly, `options`, `trackOptions` and `viewOptions`).

Reply:
I like the idea to switch to referring to the higlass `viewConfig` as `higlassConfig` everywhere it appears in the wrapper repository. But having two meanings for `viewConfig` based on the repo you are in seems more confusing than less. I think  `viewDef` and `trackDef` for view definition and track definitions, since these define the the views and tracks that higlass generates. And how is `viewDefOptions` and `trackDefOptions` since those options reside within the definition objects?

Reply:
That also makes sense that two different meanings for `viewConfig` would be confusing. I will stick to your first suggestion (i.e., `viewDef` and `trackDef`) and will leave the `options` variables as they are for now (e.g., `trackOptions`) since I prefer to use a few number of words for a variable name if possible for the readability.

Reply:
Ok that sounds good